test/test_jit.py

```diff
@@ -6443,6 +6443,8 @@ def clone_inputs(requires_grad):
     self.assertEqual(grads, grads_test)
 
     # test the grad grad case
+    if self._testMethodName in nn_functional_single_grad:
+        return
```

Comment:
Flagging @zdevito @jamesr66a. I find it hard to believe that this is the only function that only has single backwards (and not double backwards) implemented, that JIT script is exercising.